spinn_utilities/make_tools/file_converter.py

```diff
@@ -49,46 +49,43 @@
 
 class FileConverter(object):
     __slots__ = [
-        # Full destination directory
+        #: Full destination directory
```

Comment:
What does the : change?

Reply:
It turns it into a documentation comment, which you need for fields as they don't have docstrings.

I can't remember if that's in the right place or if that needs to on an assignment to the field in `__init__()`. (**TODO:** check!)

Reply:
It needs to go inside `__init__()`. Moved